openldap/server.sls

```diff
@@ -13,6 +13,8 @@
    - makedirs: True
    - require:
      - pkg: {{ openldap.server_pkg }}
+   - context:
+      openldap: {{ openldap | json }}
```

Comment:
Missing one space here (should be a four-space indent):

```suggestion
       openldap: {{ openldap | json }}
```

Reply:
done